docker/package/scripts/build-binary.sh

```diff
@@ -5,13 +5,20 @@
 set -euo pipefail
 
 export OPAMYES=true
-mkdir opamroot
+mkdir -p opamroot
```

Comment:
I'm confused by this change too.

This command creates a directory in the current working dir, how's it possible for its parent to not exist?
Did you make this to ignore the case in which `opamroot` was already there?